### PR TITLE
[framework-bundle] Fix bin/console exit code on error in Makefile

### PR DESCRIPTION
**Pull request by teohhanhui**
| Q             | A
| ------------- | ---
| License       | MIT

Do not swallow errors.

### CODE REVIEW

File: symfony/framework-bundle/3.3/Makefile [hunk 1]
```diff
@@ -1,19 +1,32 @@
+CONSOLE_BIN := $(shell which bin/console)
```
Comment (teohhanhui) [inline symfony/framework-bundle/3.3/Makefile]: False positive.

File: symfony/framework-bundle/3.3/Makefile [hunk 2]
```diff
@@ -1,19 +1,31 @@
+CONSOLE := $(shell which bin/console)
```
Comment (teohhanhui) [inline symfony/framework-bundle/3.3/Makefile]: False positive.

File: symfony/framework-bundle/3.3/Makefile [hunk 3]
```diff
@@ -1,19 +1,31 @@
...
+sf_console:
+ifndef CONSOLE
+	@printf "Run \033[32mcomposer require cli\033[39m to install the Symfony console.\n"
+endif
+
 cache-clear:
-	@test -f bin/console && bin/console cache:clear --no-warmup || rm -rf var/cache/*
+ifdef CONSOLE
+	@$(CONSOLE) cache:clear --no-warmup
```
Comment (teohhanhui) [inline symfony/framework-bundle/3.3/Makefile]: False positive?

**Comment by fabpot** [comment]
Thank you @teohhanhui.

**Pull request merged**